### PR TITLE
Make PutOOSObjectSync() thread safe

### PR DESCRIPTION
**Pull request by alxtkr77**

### CODE REVIEW

File: pkg/dataplane/http/context.go [hunk 1]
```diff
@@ -1719,11 +1719,14 @@ func (c *context) PutOOSObjectSync(putOOSObjectInput *v3io.PutOOSObjectInput) er
 		buffer.Write(ioVec)
 	}
 
-	headers := putOOSObjectHeaders
```
Comment (assaf758) [inline pkg/dataplane/http/context.go:1722]: can delete putOOSObjectHeaders?

File: pkg/dataplane/http/context.go [hunk 2]
```diff
@@ -1719,11 +1719,14 @@ func (c *context) PutOOSObjectSync(putOOSObjectInput *v3io.PutOOSObjectInput) er
...
-	headers["slice"] = strconv.Itoa(putOOSObjectInput.SliceID)
-	headers["io-vec-num"] = strconv.Itoa(len(putOOSObjectInput.Data) + 1)
-	headers["io-vec-sizes"] = iovecSizes.String()
-
+	// headers for OOS put object
+	headers := map[string]string{
+		"Content-Type":    putOOSObjectHeaders["Content-Type"],
+		"X-v3io-function": putOOSObjectHeaders["X-v3io-function"],
+		"slice":           "0",
```
Comment (assaf758) [inline pkg/dataplane/http/context.go:1726]: please check if you can remove the slice header from here and nginx - i believe the new get_slice_id() logic in v3io should make the code ignore the slice header anyway..

Comment (alxtkr77) [reply pkg/dataplane/http/context.go:1726]: will do

File: pkg/dataplane/http/context.go (hunk 1 above)
Comment (alxtkr77) [reply pkg/dataplane/http/context.go:1722]: I will break the general flow : all the headers are defined in headers.go

**Review by assaf758 — APPROVED** [review]

**Pull request merged**